### PR TITLE
lon| Amirhossein Aminian| Module-User-Focused-Data| week1

### PR DESCRIPTION
**Pull request by Amir200524**
## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: Wireframe/index.html [hunk 1]
```diff
@@ -1,13 +1,47 @@
...
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <title>What is Git?</title>
+    <link rel="stylesheet" href="styles.css">
```
Comment (SallyMcGrath) [inline Wireframe/index.html:7]:
```suggestion
    <link rel="stylesheet" href="style.css">
```

I think this will help! 😉

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Thanks for this @Amir200524 ! 

Your CSS isn't showing up because the names don't match. Edit the name and ask for someone to have another look with the CSS applied.

So far this looks like solid semantic HTML fitting the brief. Do you have anything you want help with? You have left no questions on your PR.

**Comment by Amir200524** [comment]
Good morning Sally
I hope this email finds you well.Thank you for your sudestion.I fixed it up.
Best Wishes,Amir

Yahoo Mail: Search, organise, conquer 
 
  On Sun, 13 Oct 2024 at 9:24, Sally ***@***.***> wrote:   


@SallyMcGrath requested changes on this pull request.

Thanks for this @Amir200524 !

Your CSS isn't showing up because the names don't match. Edit the name and ask for someone to have another look with the CSS applied.

So far this looks like solid semantic HTML fitting the brief. Do you have anything you want help with? You have left no questions on your PR.

In Wireframe/index.html:
>      <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    
-    
+    <title>What is Git?</title>
+    <link rel="stylesheet" href="styles.css">
⬇️ Suggested change-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="style.css">

I think this will help! 😉

**Comment by Amir200524** [comment]
Hi again
Sorry for buthering.I change my code in VS Code and pushed it on my Github and I can see my new commit on my Github but I can not see my changes in my pull request.How can I fix it?Thank you in advance for your advice.

Yahoo Mail: Search, organise, conquer 
 
  On Sun, 13 Oct 2024 at 11:13, amir ***@***.***> wrote:   Good morning Sally
I hope this email finds you well.Thank you for your sudestion.I fixed it up.
Best Wishes,Amir

Yahoo Mail: Search, organise, conquer 
 
  On Sun, 13 Oct 2024 at 9:24, Sally ***@***.***> wrote:   


@SallyMcGrath requested changes on this pull request.

Thanks for this @Amir200524 !

Your CSS isn't showing up because the names don't match. Edit the name and ask for someone to have another look with the CSS applied.

So far this looks like solid semantic HTML fitting the brief. Do you have anything you want help with? You have left no questions on your PR.

In Wireframe/index.html:
>      <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    
-    
+    <title>What is Git?</title>
+    <link rel="stylesheet" href="styles.css">
⬇️ Suggested change-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="style.css">

I think this will help! 😉

**Pull request closed without merging**